cpu/cc2538/radio/cc2538_rf_radio_ops.c

```diff
@@ -185,10 +185,9 @@ static int _read(ieee802154_dev_t *dev, void *buf, size_t size, ieee802154_rx_in
 
             /* The number of dB above maximum sensitivity detected for the
              * received packet */
-            info->rssi = -CC2538_RSSI_OFFSET + rssi_val + IEEE802154_RADIO_RSSI_OFFSET;
+            info->rssi = (CC2538_RSSI_OFFSET + rssi_val) - IEEE802154_RADIO_RSSI_OFFSET;
 
             corr_val = rfcore_read_byte() & CC2538_CORR_VAL_MASK;
-
```

Comment:
Unrelated will remove if I'm not completely wrong with the proposed changes.

Reply:
I don't think that we should be so strict about unrelated style changes. Nobody bothers to create a PR just for style changes. So if we want to improve code style, we have to stick to drive-by-changes :-)